docs/sbpy/activity/gas.rst

```diff
@@ -87,6 +100,55 @@ The gas coma models work with sbpy's apertures:
   >>> print(coma.total_number(ap))    # doctest: +FLOAT_CMP
   3.8133654170856037e+31
 
+Vectorial Model
+^^^^^^^^^^^^^^^
+
+.. warning::
+
+  Literature tests with the Vectorial model are generally in agreement at the
+  20% level or better.  The cause for the differences with the Festou FORTRAN
+  code are not yet precisely known.  Help testing this feature is appreciated.
+
```

Comment:
@sjoset I've added this warning to the documentation.  Let me know if you think that it is OK.